src/index.js

```diff
@@ -12,5 +15,7 @@ import './assets/fonts/fontello/css/fontello.css';
 
 angular
   .module('app', [adbViewModule, 'ui.router', 'pascalprecht.translate'])
+  .constant('_', _)
```

Comment:
Whoops. Yes, disregard. I assumed that lodash was already in use and it looks like it wasn't.